api/extension-guides/web-extensions.md

```diff
@@ -292,45 +292,42 @@ Check the [@vscode/test-web README](https://www.npmjs.com/package/@vscode/test-w
 
 The web bits of VS Code are downloaded to a folder `.vscode-test-web`. You want to add this to your `.gitignore` file.
 
-### Test your web extension in on vscode.dev
+### Test your web extension in vscode.dev
 
 Before you publish your extension for everyone to use on VS Code for the Web, you can verify how your extension behaves in the actual [vscode.dev](https://vscode.dev) environment.
 
 To see your extension on vscode.dev, you first need to host it from your machine for vscode.dev to download and run.
 
-From your extension's path, start an HTTP server by running `npx serve --cors -l 5000`:
+First, you'll need to [install `mkcert`](https://github.com/FiloSottile/mkcert#installation).
 
-```bash
-$ npx serve --cors -l 5000
-npx: installed 78 in 2.196s
-
-   ┌───────────────────────────────────────────────────┐
-   │                                                   │
-   │   Serving!                                        │
-   │                                                   │
-   │   - Local:            http://localhost:5000       │
-   │   - On Your Network:  http://172.19.255.26:5000   │
-   │                                                   │
-   │   Copied local address to clipboard!              │
-   │                                                   │
-   └───────────────────────────────────────────────────┘
+Then, generate the `localhost.pem` and `localhost-key.pem` files into a location you won't lose them (eg. `$HOME/certs`):
```

Comment:
prefer 'for example' rather than e.g.

Reply:
Opened a new PR for this: https://github.com/microsoft/vscode-docs/pull/6290